cli/lib/kontena/cli/helpers/health_helper.rb

```diff
@@ -49,5 +49,17 @@ def node_health(node, grid_health)
         return node['connected'] ? :ok : :offline
       end
     end
+
+    # @param node_etcd_health [Hash{health: String, error: String}]
+    # @return [Symbol, String]
+    def node_etcd_health(node_etcd_health)
+      if node_etcd_health['health']
```

Comment:
Well, it's a boolean in the API and RPC protocol schema, but I see what you mean, because the actual etcd `/health` API seriously returns `{"health": "false"}` as a string...

Not sure what to do about it though... the agent `Etcd::Health#health` validates it.

Reply:
As said: Not really an issue as there shouldn't be a case where its something else than a boolean, more like an observation of my own wft moments. :)